### PR TITLE
Adding support to register the external store

### PR DESCRIPTION
**Pull request by svt-hchaudhari**
Signed-off-by: Hemant Chaudhari <hchaudhari@hpe.com>

### Description
Adding support to register the external store

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/external_stores.py [hunk 1]
```diff
@@ -26,9 +26,20 @@
     }
 }
 
+# variable declaration
+management_ip = '10.1.188.204'
```
Comment (svt-kmcdaniel) [inline examples/external_stores.py]: Please follow this convention that has been set forth for user input
```
    "ip": "<ovc_ip>",
    "credentials": {
        "username": "<username>",
        "password": "<password>"
```

File: examples/external_stores.py [hunk 2]
```diff
@@ -26,9 +26,20 @@
...
+management_port = '9387'
+external_store_type = 'StoreOnceOnPrem'
+external_store_name = 'store1'
+username = 'svt-client'
+password = 'svt-password'
+storage_port = '9388'
+cluster_name = 'CC_Virt_00'
```
Comment (svt-kmcdaniel) [inline examples/external_stores.py]: remove environment detail

**Review by svt-kmcdaniel — CHANGES_REQUESTED** [review]
Please remove test environment setting(s)

File: examples/external_stores.py (hunk 1 above)
Comment (svt-hchaudhari) [reply examples/external_stores.py]: Updated as per the format

File: examples/external_stores.py (hunk 2 above)
Comment (svt-hchaudhari) [reply examples/external_stores.py]: removed the details

File: simplivity/resources/external_stores.py [hunk 3]
```diff
@@ -77,6 +78,38 @@ def get_by_data(self, data):
...
+            password: The client password of the external store
+            management_port: The management IP port of the external store. Default: 9387
+            storage_port: The storage IP port of the external store. Default: 9388
+            external_store_type: The type of external store. Default: StoreOnceOnPrem
+            timeout: Time out for the request in seconds.
+
+        Returns:
+            object: External store object.
+        """
+        resource_uri = "{}".format(URL)
```
Comment (sijeesh) [inline simplivity/resources/external_stores.py]: Is the above line required?

Comment (svt-hchaudhari) [reply simplivity/resources/external_stores.py]: removed the line, this is not required.

**Review by sijeesh — APPROVED** [review]

**Pull request merged**